### ISSUE TITLE
React-native fails to install after Mac Monterey update

### ISSUE THREAD

**Issue by spiroalisandratos** [react-native-community/cli#1631]
## Environment

Mac OS Monterey - Any React Native release

## Description

/usr/local/lib/node_modules/react-native-cli/index.js:302
  cli.init(root, projectName);
      ^

## Reproducible Demo

react-native init projectName

**Comment by cortinico** [comment]
Looks like a duplicate of this other issue (check my suggestion https://github.com/facebook/react-native/issues/34055#issuecomment-1165887713)

**Comment by spiroalisandratos** [comment]
Yep it is the same issue, however the solution has not worked and the case was closed so I opened this one.  I also updated watchman and gave perms to it.  Once all this was done the error was still present and this particular project is a fresh install just to test react native followed directly from the react native site.  Never had issue prior to Mac upgrade..

**Comment by spiroalisandratos** [comment]
Disabled SIP on Mac and issue goes away.  Will need to understand what the new Mac upgrade is blocking.  Making mods to react native packages, react native global is also not an issue here.
